gatsby/src/templates/situation/list.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import { IArea } from 'graphql-types';
+import Container from '@/components/container';
+import Headline from '@/components/headline';
+import ListCard from '@/components/list-card';
+
+interface IQueryResult {
+  area?: IArea;
+}
+
+interface IProps {
+  data: IQueryResult;
+}
+
+const Home: React.FC<IProps> = ({ data }) => {
+  const { area } = data;
+  return (
+    <>
+      <Container>
+        <Headline>{area.name}</Headline>
+        <div>
+          {area.relationships.situation &&
+            area.relationships.situation.map((item, index) => {
```

Comment:
```suggestion
            area.relationships.situation?.map((item, index) => {
```
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html